src/bespokelabs/curator/request_processor/openai_batch_request_processor.py

```diff
@@ -765,13 +825,23 @@ def track_already_downloaded_batches(self):
             glob.glob(f"{self.working_dir}/batch_objects_downloaded_*.jsonl")
         )
         for downloaded_batch_object_file in downloaded_batch_object_files:
+            logger.info(
+                f"Processing downloaded batch objects file: {downloaded_batch_object_file} Your API key is ***{self.client.api_key[-4:]}."
+            )
             with open(downloaded_batch_object_file, "r") as f:
                 for line in f:
                     batch_object = Batch.model_validate(json.loads(line))
                     request_file = batch_object.metadata["request_file_name"]
                     response_file = request_file_to_response_file(request_file, self.working_dir)
-                    assert request_file in self.tracker.unsubmitted_request_files
-                    assert os.path.exists(response_file)
+                    assert (
+                        request_file in self.tracker.unsubmitted_request_files
+                    ), f"request_file {request_file} not in unsubmitted_request_files: {self.tracker.unsubmitted_request_files}"
+                    if not os.path.exists(response_file):
+                        logger.warning(
+                            f"response_file {response_file} does not exist. will continue..."
```

Comment:
Again, say that this file doesn't exist during tracking of downloaded batch files and will resubmit.